terraform/spokes/variables.tf

```
@@ -16,6 +16,9 @@ variable "addons" {
     enable_metrics_server               = true
     enable_karpenter                    = true
     enable_cw_prometheus                = true
+    enable_kyverno                      = true
+    enable_kyverno_policy_reporter      = true
+    enable_kyverno_policies             = true    
```

```suggestion
    enable_kyverno_policies             = true    
```
```suggestion
    enable_kyverno_policy_reporter      = true
    enable_kyverno_policies             = true    
```
Can you activate these 2 addons here also
- https://github.com/aws-samples/fleet-management-on-amazon-eks-workshop/blob/riv24/gitops/addons/clusters/fleet-spoke-prod/addons/gitops-bridge/values.yaml
- https://github.com/aws-samples/fleet-management-on-amazon-eks-workshop/blob/riv24/gitops/addons/clusters/fleet-spoke-staging/addons/gitops-bridge/values.yaml